src/GraphQL/ObjectExtensions.cs

```diff
@@ -36,6 +34,13 @@ public static T ToObject<T>(this IDictionary<string, object> source)
         /// </param>
         public static object ToObject(this IDictionary<string, object> source, Type type, IGraphType mappedType = null)
         {
+            if (source == null)
```

Comment:
I have doubts about the need for this check.

Reply:
In any case, it won’t get any worse, since before this check it would be NRE later